### ISSUE TITLE
Rule `svelte/no-dom-manipulating` throws errors when reading properties

### ISSUE THREAD

**Issue by robinloeffel** [ota-meshi/eslint-plugin-svelte#378]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.34.0

### What version of `eslint-plugin-svelte` are you using?

2.19.0

### What did you do?

With the following snippet, the rule [`svelte/no-dom-manipulating`](https://ota-meshi.github.io/eslint-plugin-svelte/rules/no-dom-manipulating/) throws the following error:

```js
let mobileExpandContent: HTMLDivElement;
let mobileMenuExpandedHeight = "";

// ...

mobileMenuExpandedHeight = `${mobileExpandContent.clientHeight}px`;

// ...

<div bind:this={mobileExpandContent}></div>
```

```
Don't manipulate the DOM directly. The Svelte runtime can get confused if there is a difference between the actual DOM and the DOM expected by the Svelte runtime.eslint[svelte/no-dom-manipulating](https://ota-meshi.github.io/eslint-plugin-svelte/rules/no-dom-manipulating/)
```

The rule is configured to simply `"svelte/no-dom-manipulating": "error"`.

### What did you expect to happen?

I would not expect the rule to go off when only reading DOM properties.

### What actually happened?

The rule went off.

### Link to **GitHub Repo** with Minimal Reproducible Example

n/a; You can reproduce it on the [playground of the rule](https://ota-meshi.github.io/eslint-plugin-svelte/rules/no-dom-manipulating/) by trying to access the `clientHeight` of one of the referenced DOM nodes.
